bokehjs/src/coffee/api/models.coffee

```diff
@@ -81,20 +81,7 @@ module.exports = {
   LinearColorMapper:                      require("../models/mappers/linear_color_mapper").Model
 
   ## api/typings/models/markers.d.ts
-  Marker:                                 require("../models/markers/marker").Model
-  Asterisk:                               require("../models/markers/asterisk").Model
-  Circle:                                 require("../models/glyphs/circle").Model
-  CircleCross:                            require("../models/markers/circle_cross").Model
-  CircleX:                                require("../models/markers/circle_x").Model
-  Cross:                                  require("../models/markers/cross").Model
-  Diamond:                                require("../models/markers/diamond").Model
-  DiamondCross:                           require("../models/markers/diamond_cross").Model
-  InvertedTriangle:                       require("../models/markers/inverted_triangle").Model
-  Square:                                 require("../models/markers/square").Model
-  SquareCross:                            require("../models/markers/square_cross").Model
-  SquareX:                                require("../models/markers/square_x").Model
-  Triangle:                               require("../models/markers/triangle").Model
-  X:                                      require("../models/markers/x").Model
+  markers:                                [require('../models/markers/all_markers')]
```

Comment:
Ah yes better name thanks 

> On Aug 23, 2016, at 03:37, Mateusz Paprocki <notifications@github.com> wrote:
> 
> In bokehjs/src/coffee/api/models.coffee:
> 
> > @@ -81,20 +81,7 @@ module.exports = {
> >    LinearColorMapper:                      require("../models/mappers/linear_color_mapper").Model
> >  
> >    ## api/typings/models/markers.d.ts
> > -  Marker:                                 require("../models/markers/marker").Model
> > -  Asterisk:                               require("../models/markers/asterisk").Model
> > -  Circle:                                 require("../models/glyphs/circle").Model
> > -  CircleCross:                            require("../models/markers/circle_cross").Model
> > -  CircleX:                                require("../models/markers/circle_x").Model
> > -  Cross:                                  require("../models/markers/cross").Model
> > -  Diamond:                                require("../models/markers/diamond").Model
> > -  DiamondCross:                           require("../models/markers/diamond_cross").Model
> > -  InvertedTriangle:                       require("../models/markers/inverted_triangle").Model
> > -  Square:                                 require("../models/markers/square").Model
> > -  SquareCross:                            require("../models/markers/square_cross").Model
> > -  SquareX:                                require("../models/markers/square_x").Model
> > -  Triangle:                               require("../models/markers/triangle").Model
> > -  X:                                      require("../models/markers/x").Model
> > +  markers:                                [require('../models/markers/all_markers')]
> node.js-idiomatic module name is markers/index.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>